### PR TITLE
Grap 24 SignIn & SignUp Forms

### PR DESCRIPTION
**Pull request by 1LisaW**
Created forms with mocked data handlers. Snapshot tests for impemented components

### CODE REVIEW

File: src/components/FormInput/FormInput.test.tsx
Comment (ppmarkek) [inline src/components/FormInput/FormInput.test.tsx:1]: Create new folder for test

File: src/pages/SignIn/SignIn.tsx
Comment (ppmarkek) [inline src/pages/SignIn/SignIn.tsx:1]: Create new folder for test

File: src/pages/SignIn/SignIn.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      email: '',
+      password: '',
+      passwordConfirm: '',
+    },
+    onSubmit: (values) => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+  return (
+    <Grid
```
Comment (ppmarkek) [inline src/pages/SignIn/SignIn.tsx:19]: Use 'styled mui' for component styling

File: src/pages/SignUp/SignUp.tsx
Comment (ppmarkek) [inline src/pages/SignUp/SignUp.tsx:1]: Create new folder for test

File: src/pages/SignUp/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      email: '',
+      password: '',
+      passwordConfirm: '',
+    },
+    onSubmit: (values) => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+  return (
+    <Grid
```
Comment (ppmarkek) [inline src/pages/SignUp/SignUp.tsx:19]: Use 'styled mui' for component styling

**Review by svyatoslavkk — APPROVED** [review]

**Review by ppmarkek — APPROVED** [review]

**Pull request merged**